static/js/portico/help.js

```diff
@@ -65,14 +65,10 @@ function render_code_sections() {
     });
 }
 
-function scrollToHash(simplebar) {
-    const hash = window.location.hash;
-    const scrollbar = simplebar.getScrollElement();
-    if (hash !== "" && $(hash).length > 0) {
-        const position = $(hash).position().top - $(scrollbar.firstChild).position().top;
-        scrollbar.scrollTop = position;
-    } else {
-        scrollbar.scrollTop = 0;
+function scroll_top_if_empty_hash(ele) {
```

Comment:
If the page content is updated and the hash remains empty, this function scrolls the page to the top. If there is any hash in the link, it is automatically handled by the browser scrollbar. So, no need to add that case.